geometry_msgs/Pose objPose
string objLabel
